docker/hive/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # trunk-ignore-all(terrascan/AC_DOCKER_0047)
 
 # CairoVM Chain
-FROM ghcr.io/dojoengine/dojo:v1.0.0-alpha.15 as katana
+FROM ghcr.io/dojoengine/dojo:v1.0.0-alpha.19 as katana
```

Comment:
did you check that this works?

Reply:
Didn't checked yet but the CI should confirm no?